Automattic/wp-calypso
# Redesign My Home domain upsell as top banner

Fixes https://github.com/Automattic/dotcom-forge/issues/1911

## Proposed Changes

Move the My Home Domain Upsell to the top carousel tasks.
Design discussion2: pebzTe-RL-p2 / p1678475998639649-slack-C03FE3T479R
Figma: LruC8ztBCLJMFNkQN1JNm3-fi-2025%3A17367

## Tasks
- [x] Remove the old My Home Upsell
- [x] Remove the old domain upsell from the carousel
- [x] Update the final images.
- [x] Remove unit tests tests for my home upsell
- [x] Remove e2e tests from my home upsell
- [x] Add new unit tests
- [ ] Add new e2e tests (follow-up: https://github.com/Automattic/dotcom-forge/issues/1983)
- [x] Add `lock` icon to the mobile suggested domain 🔒 
- [ ] Confirm which track events should track (follow-up: https://github.com/Automattic/dotcom-forge/issues/1982)

## Screenshot 
#### Desktop
| Yearly plans | Free/Monthly plans |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/402286/225400804-a64feb93-c123-475a-9f87-a689c40846c8.png) | ![image](https://user-images.githubusercontent.com/402286/225400825-ecdc1459-f728-4298-b893-36d579ea3128.png) |

#### Mobile
| Yearly plans | Free/Monthly plans |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/402286/225428070-1a969e8c-5d0f-4b0f-9377-7314abc22925.png) | ![image](https://user-images.githubusercontent.com/402286/225427834-6e78d128-a14e-40ef-9338-8771de7083af.png) |

## Testing Instructions
- You should apply this diff: D104724-code
- Using a launched site without custom domains, go to `/home`
- You should see the new Domain Upsell

## Pre-merge Checklist
